python/plugins/processing/algs/qgis/QgisAlgorithmProvider.py

```diff
@@ -75,7 +75,6 @@
 from .RasterCalculator import RasterCalculator
 from .RasterLayerHistogram import RasterLayerHistogram
 from .RasterSampling import RasterSampling
-from .RectanglesOvalsDiamondsFixed import RectanglesOvalsDiamondsFixed
```

Comment:
Nice move!
You need to remove the help from the YML too.

Reply:
Which help?

Reply:
This file https://github.com/qgis/QGIS/blob/master/python/plugins/processing/algs/help/qgis.yaml
But indeed, I can't see the alg, so it means that the help was missing before?

Reply:
I know about this file, but it contains nothing related to these two algs, that's why I asked.

Reply:
Ok!  I can't resolve the comment.